scripts/format.bash

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
Why do we now have a `format.sh` and `shfmt.sh` with the same contents except `--write` flag?

Reply:
`shfmt.bash` is what we use in CI and just prints incorrect formatting

I introduced `format.bash` as a script to be used locally to apply format changes. Previously, we had to manually fix the shfmt errors, or rely on people utilising editorconfig or shfmt IDE plugins to perform formatting. This makes it easier for us to blanket-apply changes.

Reply:
Excellent! Thank you @jthegedus !